pkg/gui/controllers/helpers/refresh_helper.go

```diff
@@ -638,12 +640,12 @@ func (self *RefreshHelper) refreshReflogCommits() error {
 		return nil
 	}
 
-	if err := refresh(&model.ReflogCommits, "", ""); err != nil {
+	if err := refresh(&model.ReflogCommits, "", "", ""); err != nil {
 		return err
 	}
 
 	if self.c.Modes().Filtering.Active() {
-		if err := refresh(&model.FilteredReflogCommits, self.c.Modes().Filtering.GetPath(), self.c.Modes().Filtering.GetAuthor()); err != nil {
+		if err := refresh(&model.FilteredReflogCommits, self.c.Modes().Filtering.GetPath(), self.c.Modes().Filtering.GetAuthor(), self.c.Modes().Filtering.GetChanges()); err != nil {
```

Comment:
Even here I'd just go `self.c.Modes().Filtering.GetFilters()` and have that return a CommitFilters struct.